lib/compile.coffee

```diff
@@ -163,6 +163,7 @@ unescapeTripleBraces = (str) ->
 
 compile = (options) ->
   filePath = argv.file or options.file
+  filePath = filePath?.replace /^\//, '' # replace leading / to read the file
```

Comment:
are you sure we need to do this here? I could see a script passing an absolute path in bc we are reading files - e.g. node often times wants to work in absolute paths so getting a path to a file often results in `/Users/steve/...` in which case we want that preceeding slash

All the other places make sense to me though as those are in ng-includes, but this one is just different in that it is passed directly to fs.readFileSync